website/addons/gitlab/templates/gitlab_user_settings.mako

```diff
@@ -0,0 +1,45 @@
+<!-- Authorization -->
+<div id='gitlabAddonScope' class='addon-settings addon-generic scripted'
+     data-addon-short-name="${ addon_short_name }"
+     data-addon-name="${ addon_full_name }">
+
+    <%include file="gitlab_credentials_modal.mako"/>
+
+    <h4 class="addon-title">
+        <img class="addon-icon" src=${addon_icon_url}>
+        <span data-bind="text: properName"></span> <!-- TODO: Can we use mako addon_full_name as some other addons do? -->
+        <small>
+            <a href="#gitlabInputCredentials" data-toggle="modal" class="pull-right text-primary">Connect Account</a>
```

Comment:
Minor: This language recently changed from `Connect Account` to `Connect or Reauthorize Account`